web/css/notifications.css

```diff
@@ -4,8 +4,8 @@
   position: relative;
 }
 
-#wv-info-button label::before {
-  content: attr(data-content) '';
```

Comment:
Why did the hyphen get removed? `data-*` is the standard HTML attribute syntax.

Reply:
I didn't think `react` hyphens with attributes. I was wrong. Good call 👍